helm/dex-app/templates/dex-secret.yaml

```diff
@@ -32,6 +32,11 @@ stringData:
       - https://{{ .Values.Installation.V1.GiantSwarm.OIDC.ClientAddress }}/callback
       name: dex-k8s-authenticator
       secret: {{ .Values.Installation.V1.Secret.OIDC.ClientSecret }}
+    - id: zQiFLUnrTFQwrybYzeY53hWWfhOKWRAU
+      redirectURIs:
+      - http://localhost:8085/oauth/callback
+      name: gscliauth
+      secret: {{ .Values.Installation.V1.Secret.OIDC.ClientSecret }}
```

Comment:
Can this be the same as for the other clients?

Reply:
I'm not fully sure. Ping @giantswarm/sig-security

Reply:
it would be best to have a separate one for each client. As a general rule of security, do not share secrets between users/clients.

Reply:
Thank you, I've updated it 🚀